shotover-proxy/src/server.rs

```diff
@@ -437,13 +447,13 @@ impl<C: Codec + 'static> Handler<C> {
                             }
                         },
                         Err(_) => {
-                            if idle_time_seconds < 35 {
-                                trace!("Connection Idle for more than {} seconds {}",
-                                    idle_time_seconds, self.conn_details);
-                            } else {
-                                debug!("Dropping. Connection Idle for more than {} seconds {}",
-                                    idle_time_seconds, self.conn_details);
-                                return Ok(());
+                            if let Some(timeout) =  self.timeout {
+                                if idle_time_seconds < timeout {
+                                    trace!("Connection Idle for more than {} seconds {}", timeout, self.conn_details);
```

Comment:
seems wrong to have a louder trace level for not hitting the timeout than actually hitting the timeout?

Reply:
Oh it was always like that...

Reply:
maybe just drop it to `debug!` level while we are here?